src/commands/search_tx_optimizations.rs

```diff
@@ -0,0 +1,93 @@
+use anyhow::{bail, Result};
+use clap::Args;
+use sqlx::{PgPool, Row};
+
+#[derive(Debug, Args)]
+#[command(about = "Command to apply or drop search transaction optimizations in the archive database.")]
+pub struct SearchTxOptimizationsCommand {
+  /// URL to the archive database
+  #[arg(long, env = "MINAMESH_ARCHIVE_DATABASE_URL")]
+  archive_database_url: String,
+
+  /// Apply optimizations
+  #[arg(long, conflicts_with = "drop")]
+  apply: bool,
+
+  /// Drop optimizations
+  #[arg(long, conflicts_with = "apply")]
+  drop: bool,
+
+  /// Check if optimizations are applied
+  #[arg(long)]
+  check: bool,
+}
+
+impl SearchTxOptimizationsCommand {
+  pub async fn run(&self) -> Result<()> {
+    // Connect to the database
+    let pool = PgPool::connect(&self.archive_database_url).await?;
+
+    // Check if optimizations are already applied
+    if self.apply && self.check_if_optimizations_applied(&pool).await? {
+      bail!("Search transaction optimizations are already applied.");
+    } else if self.drop && !self.check_if_optimizations_applied(&pool).await? {
+      bail!("Search transaction optimizations are not applied.");
+    }
+
+    if self.apply {
+      self.apply_optimizations(&pool).await?;
+    } else if self.drop {
+      self.drop_optimizations(&pool).await?;
+    } else if self.check {
+      let applied = self.check_if_optimizations_applied(&pool).await?;
+      if applied {
+        println!("Search transaction optimizations are already applied.");
+      } else {
+        println!("Search transaction optimizations are not applied.");
+      }
+    } else {
+      bail!("You must specify either --apply or --drop or --check.");
+    }
+    Ok(())
+  }
+
+  async fn apply_optimizations(&self, pool: &PgPool) -> Result<()> {
+    println!("Applying search transaction optimizations on Archive Database (this may take few minutes)...");
+
+    // Load and execute the SQL from the file
+    let sql = include_str!("../../sql/migrations/apply_search_tx_optimizations.sql");
```

Comment:
For cases like this, we should probably use the following (incase we want to move the file elsewhere / refactor the line elsewhere, we don't need to worry about the relative reference)

```rs
include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/root-relative/path/to/the/file.sql"))
```